x-pack/plugins/index_management/__jest__/a11y/indices_tab.a11y.test.ts

```diff
@@ -0,0 +1,129 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { act } from 'react-dom/test-utils';
+// import { expectToBeAccessible } from '@kbn/test-jest-helpers';
+import { getA11yViolations } from '@kbn/test-jest-helpers';
+import { IndicesTestBed, setup } from '../client_integration/home/indices_tab.helpers';
+import {
+  indexMappings,
+  indexSettings,
+  indexStats,
+  setupEnvironment,
+} from '../client_integration/helpers';
+import {
+  createDataStreamBackingIndex,
+  createNonDataStreamIndex,
+} from '../client_integration/home/data_streams_tab.helpers';
+
+describe('A11y Indices tab', () => {
+  let testBed: IndicesTestBed;
+  let server: ReturnType<typeof setupEnvironment>['server'];
+  let httpRequestsMockHelpers: ReturnType<typeof setupEnvironment>['httpRequestsMockHelpers'];
+
+  beforeEach(async () => {
+    ({ server, httpRequestsMockHelpers } = setupEnvironment());
+    httpRequestsMockHelpers.setLoadIndicesResponse([]);
+    testBed = await setup();
+  });
+
+  afterAll(() => {
+    server.restore();
+  });
+
+  it('when there are no indices', async () => {
+    httpRequestsMockHelpers.setLoadIndicesResponse([]);
+    await act(async () => {
+      testBed = await setup();
+    });
+    const { component } = testBed;
+    component.update();
+    // this is expected to fail and needs to be updated when eui#5674 is available in Kibana
```

Comment:
👍🏼 Wondering if we should perhaps create a new issue to update this bit of code when this eui issue is fixed

Reply:
I changed the comment to mention that the problem will be resolved with EUI 52.0.0 update (probably in this [PR](https://github.com/elastic/kibana/pull/128313)). I think we don't need an extra issue for that because when EUI version is updated, the tests will start to fail and will need to be updated as well.